### DISCUSSION TITLE
0.10.1

### DISCUSSION THREAD

**Discussion by Ashinch** [ReadYouApp/ReadYou#737] [Announcements]
## 0.10.1

1.	Fix text icon style for feed (#726, @Ashinch)
2.	Fix issue where early users couldn’t launch the app after importing app preferences (#718, @Ashinch)
3.	Fix some cases causing OPML import failures (#735, @Ashinch)
4.	Fix some window navigation animations (#717, @JunkFood02)
5.	Fix some style issues for RTL languages (#732, #733, @JunkFood02)
6.	Translation updates (Thanks to all contributors)

### Notes

**Full changelog:** [0.10.0...0.10.1](https://github.com/Ashinch/ReadYou/compare/0.10.0...0.10.1)

This is a bug fix patch. In case you missed it, here is the changelog for version 0.10.0:

## 0.10.0

1. Bump up Material Design 3 dependencies, compile sdk, and gradle version (#502, @JunkFood02)
2. Support the in-app language picker for Android 13 (#541, #571, @JunkFood02)
3. Support add feed via system share sheet (#618, @JunkFood02)
4. Support for specifying the composition of shared content (#660, @Ashinch)
5. Improve swipe to star/unstar, swipe to unread, and add configuration (#594, @JunkFood02)
6. Use system locale to format time display by default. (#617, @JunkFood02)
7. Switch to androidx edge to edge implementation  (#690, @Moderpach)
8. Add image viewer to the reading page (#578, #545, @JunkFood02, @nvllz)
9. Add gestures to swipe up and down on the reading page to switch articles (#589, @JunkFood02)
10. Add crash report activity to handle uncaught exceptions (#576, @JunkFood02)
11. Add context menu on long press for items in the flow page (#613, @JunkFood02)
12. Add line height multiple preference for reading page (#620, @JunkFood02)
13. Add license list page and improve about page (#664, @Ashinch)
14. Add troubleshooting page and app preferences import/export tools (#672, #710, @Ashinch)
15. Fix notification permission request at startup on Android 13 (@JunkFood02)
16. Fix clash when published date and updated date are both empty (@JunkFood02)
17. Now replace publish date of an article with the current time if it is a future date (#638, @Ashinch)
18. Now when cleaning feed or group, the starred articles will be ignored (#652, @Ashinch)
19. Now automatically restart the app after loading external fonts (#667, @Ashinch)
20. Now first find the `<enclosure>` tag as a thumbnail of the article (#681, @Ashinch)
21. Now by default, sort categories alphabetically during sync on Google Reader (#700, @mbestavros)
22. New image placeholder (#712, @JunkFood02)
23. More UI improvements and bug fixes (@Ashinch, @JunkFood02)
24. Translations updates (thanks to everyone who helped out)

### Notes

1. In order to support the export of setting items, some of the setting items have been reset to default and you may have to reset them.
2. The buymeacoffee.com sponsorship channel is currently unavailable, and the money previously donated has been refunded to the donors' accounts.
3. We are looking for other sponsorship channels, currently available are AFDIAN. If you would like to support the development of ReadYou, you can sponsor it through [AFDIAN](https://afdian.net/a/ashinch).
4. Thanks to our sponsors: @User_3072e, @User_223be, @User_3c5c7(Simon), @User_97ebe, @Lowae, @User_28b9f, @User_yuHC, @nullqwertyuiop, @openisgood, @User_vWca, @qgmzmy, @User_97ee1

**Full changelog:** [0.9.12...0.10.0](https://github.com/Ashinch/ReadYou/compare/0.9.12...0.10.0)

<hr /><em>This discussion was created from the release <a href='https://github.com/Ashinch/ReadYou/releases/tag/0.10.1'>0.10.1</a>.</em>

**Comment by Zinoujoker** [comment]
Good keep up 🆙 👍 😅
